drivers/st77xx/Makefile.dep

```diff
@@ -0,0 +1,12 @@
+FEATURES_REQUIRED += periph_spi
+FEATURES_REQUIRED += periph_gpio
+
+USEMODULE += lcd
+USEMODULE += lcd_multi_cntrl
+USEMODULE += ztimer
+USEMODULE += ztimer_msec
+
+ifeq (,$(filter st7789 st7796,$(USEMODULE)))
+  # enable st7735 as default if no other module is enabled
+  USEMODULE += st7735
+endif
```

Comment:
This block should go in `drivers/Makefile.dep`

Reply:
I don't like that the `drivers/Makefile.dep` is blown up more an more with driver specific dependencies. The goal should be to keep `drivers/Makefile.dep` as simple and small as possible and to resolve driver specific dependencies in the driver implementation. That was a further reason why I selected `st77xx` instead of `st77*` when  `displ_dev` is enabled. In that case, the driver's `Makefile.dep' is used to resolve the driver specific dependencies.